lib/cisco_node_utils/cmd_ref/feature.yaml

```diff
@@ -35,9 +35,10 @@ nv_overlay_evpn:
 
 vn_segment_vlan_based:
   # MT-lite only
-  /N(3|9)/:
+  N3k: &vn_segment_vlan_based_N3k
     kind: boolean
     config_get: 'show running section feature'
     config_get_token: '/^feature vn-segment-vlan-based$/'
     config_set: 'feature vn-segment-vlan-based'
     default_value: false
+  N9k: *vn_segment_vlan_based_N3k
```

Comment:
Great suggestion. Updated both README_YAML and develop-best-practices with this information. Mind taking a look?